### PR TITLE
Make Travis CI to report failures on Node.js 4.0

### PR DESCRIPTION
**Pull request by gdi2290**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ node_js:
   - "iojs-v1"
   - "iojs-v2"
   - "iojs-v3"
+  - "4.0"
```
Comment (mzgol) [inline .travis.yml]: It should be `4`, not `4.0`. Node will surely get a lot of minor updates before it goes LTS and node-sass should test on the latest of them.

This is analogous to how in the above lines you can see `iojs-v3`, not `iojs-v3.0`.

Comment (gdi2290) [inline .travis.yml]: good point, one sec I'll update the pr

**Comment by gdi2290** [comment]
@mzgol  I pushed up the change thanks for your feedback

**Comment by mzgol** [comment]
:+1:

**Comment by xzyfer** [comment]
Thanks @gdi2290 and @mzgol. We'll merge this when we ship Node.js 4 support

**Pull request merged**